ReleaseNotes.md

```diff
@@ -17,6 +17,13 @@
   (#3696)
 - There have been numerous changes to the RGroup Decomposition code which change
   the results. (#3767)
+- In RGroup Decomposition, when onlyMatchAtRGroups is set to false, each molecule
+  is now decomposed based on the first scaffold which adds/uses the least number
+  of non-user-provided R labels, rather than the first scaffold tout court.
```

Comment:
“tout court”?

Reply:
I thought that was international, but maybe not. Changed :)